ldapadmin/src/main/webapp/console/app/components/stats/stats.es6

```diff
@@ -67,13 +67,42 @@ class StatsController {
             return value
           }
         }
+
       }
     }
+    let el = $element.find('.chartist')
     this.lines = new Chartist[this.type=='bar' ? 'Bar' : 'Line'](
-      $element.find('.chartist')[0], this.parsed, options
+      el[0], this.parsed, options
+    )
+    this.lines.on('draw', (data) => {
+      if(data.type === 'label') {
+        let ydiff = 8
+        if (data.axis.units.dir == 'vertical') {
+          let delta = el.height() / (data.axis.ticks.length)
+          ydiff = (this.type == 'bar') ? 18 : ( delta )
+        }
+        let text = Chartist.Svg('text', {
+          x : data.x,
+          y : data.y + ydiff
+        }).text(data.text)
+        data.element.replace(text)
+      }
+    })
+    this.$injector.get('translate')(this.title).then(
+      (v) => el.attr('title', v)
     )
     this.view = 'graph'
 
+    this.export = () => {
+      let el = $element.find('svg')
+      el.append($(
+        '<style>' +
+        Array.from(document.styleSheets[3].cssRules).map(x => x.cssText).join('') +
```

Comment:
'3' reference the third stylesheet ? Maybe you can use an identifier, because if somebody add stylesheets before `svg.css`, he also need to update this index.